internal/flypg/pg.go

```diff
@@ -169,6 +171,43 @@ func (c *PGConfig) SetDefaults() error {
 		"wal_log_hints":            true,
 		"hot_standby":              true,
 		"shared_preload_libraries": fmt.Sprintf("'%s'", strings.Join(sharedPreloadLibraries, ",")),
+		"restore_command":          "",
+		"recovery_target":          "",
+		"recovery_target_timeline": "",
+		"recovery_target_action":   "",
```

Comment:
if these are empty (i.e. CLOUD_ARCHIVING_WAL_RESTORE is not set), postgres fails to start with a syntax error

Reply:
Good catch.